playbooks/templates/docker-compose.override.yml.j2

```diff
@@ -0,0 +1,11 @@
+version: "3.7"
+
+services:
+  sia:
+    build:
+      args:
+        branch: "{{ portal_skyd_version }}"
+{% if accounts_on and (portal_accounts_version | default('') != '' or portal_accounts_image_version | default('') != '') %}
+  accounts:
+    {{ docker_compose_override_data.services.accounts | to_nice_yaml(width=255, indent=2) | indent(4) }}
+{% endif %}
```

Comment:
We still will be able to override the image for the website right? I just don't see that part here anywhere.